queue_job/jobrunner/channels.py

```diff
@@ -410,11 +419,13 @@ def configure(self, config):
 
         * capacity
         * sequential
+        * subcapacity
         * throttle
         """
         assert self.fullname.endswith(config["name"])
         self.capacity = config.get("capacity", None)
         self.sequential = bool(config.get("sequential", False))
+        self.subcapacity = int(config.get("subcapacity", 0)) or None
```

Comment:
I perceive this `0 or None` falsy coercion as contrived. I'm open to a better idiom.